Description: Removes #include of header file has been removed.
Origin: upstream, https://bugs.eclipse.org/bugs/show_bug.cgi?id=272373
Applied-Upstream: Yes

diff --git a/results/plugins/org.eclipse.cdt.core.linux/library/openpty.c b/results/plugins/org.eclipse.cdt.core.linux/library/openpty.c
index 3a86072..9747c7f 100644
--- a/results/plugins/org.eclipse.cdt.core.linux/library/openpty.c
+++ b/results/plugins/org.eclipse.cdt.core.linux/library/openpty.c
@@ -20,7 +20,6 @@
 #include <grp.h>
 
 #include <stdlib.h>
-#include <stropts.h>
 
 /**
  * This is taken from R. W. Stevens book.
